### PR TITLE
Leap year

### PR DESCRIPTION
**Pull request by tbhaxor**
**Fixes issue:**

**Changes:** Added leap year program in Javascript

### CODE REVIEW

File: code/unclassified/src/leap_year/leap_year.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Part of cosmos from opengenus foundation
+
+const isLeep = year => ((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0)
+
+
```
Comment (arnavb) [inline code/unclassified/src/leap_year/leap_year.js]: Remove extra newline. Also call it `isLeap`.

File: code/unclassified/src/leap_year/leap_year.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+var startYear = 1990
```
Comment (arnavb) [inline code/unclassified/src/leap_year/leap_year.js]: Use let.

File: code/unclassified/src/leap_year/leap_year.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+var endYear = 2020
+
+for(i = startYear; i<= endYear; i++) {
```
Comment (arnavb) [inline code/unclassified/src/leap_year/leap_year.js]: `for (let i = ...)`

File: code/unclassified/src/leap_year/leap_year.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// Part of cosmos from opengenus foundation
```
Comment (arnavb) [inline code/unclassified/src/leap_year/leap_year.js]: This comment isn't necessary.

File: code/unclassified/src/leap_year/leap_year.js (hunk 1 above)
Comment (tbhaxor) [reply code/unclassified/src/leap_year/leap_year.js]: sorry. typing mistook :laughing:

File: code/unclassified/src/leap_year/leap_year.js (hunk 4 above)
Comment (tbhaxor) [reply code/unclassified/src/leap_year/leap_year.js]: next time i ll take care :+1:

File: code/unclassified/src/leap_year/leap_year.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+
+const isLeap = year => ((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0)
+
+let startYear = 1990
+let endYear = 2020
+
+for(let i = startYear; i<= endYear; i++) {
```
Comment (arnavb) [inline code/unclassified/src/leap_year/leap_year.js]: One more change. Add a space before the opening parentheses here and before the first one on the line below. Additionally, use pre-increment over post increment.

**Pull request closed without merging**